setup.py

```diff
@@ -71,7 +71,7 @@
             "pytest-cov >=2.8.1, ==2.8.*",
             "pytest-mock >= 2.0.0, ==2.0.*",
             "recommonmark >=0.5.0, ==0.*",
-            "snakemake >=5.4.0, ==5.*",
+            "snakemake >=5.4.0, <5.27",
```

Comment:
We shouldn't need this additional constraint anymore. [They changed the minimum required version of Snakemake back to 3.5](https://github.com/snakemake/snakemake/commit/e025b98d76e9565b73cadfca0bb3909ed869aaea) in the release they made today (5.27.4). This should fix [the error Travis CI originally encountered](https://travis-ci.com/github/nextstrain/augur/jobs/426156466#L2361).

Reply:
ok!

Reply:
Darn, it looks like there are still bugs with version <3.7 and the newest Snakemake. [There is an issue open for the specific error we see in CI](https://github.com/snakemake/snakemake/issues/725), but the easiest fix for us is the original constraint you had here. Sorry about that!